src/js/views/channel_slider_view.js

```diff
@@ -104,11 +104,12 @@ var ChannelSliderView = Backbone.View.extend({
                 var endAvg = parseInt(ends.reduce(addFn, 0) / ends.length, 10);
                 var startsNotEqual = starts.reduce(allEqualFn, starts[0]) === false;
                 var endsNotEqual = ends.reduce(allEqualFn, ends[0]) === false;
-                var min = mins.reduce(reduceFn(Math.min), 9999);
-                var max = maxs.reduce(reduceFn(Math.max), -9999);
+                var min = mins.reduce(reduceFn(Math.min));
+                var max = maxs.reduce(reduceFn(Math.max));
+                console.log(min, max);
```

Comment:
@joshmoore it's perfectly true that logs for development and debugging purposes should of course be taken out (for production) but the stackoverflow topic as well as other discussions are all based on what I would call an obsolete condemnation of the use of cosole.xxx methods for logging. They used to not be implemented by all browsers and/or undefined (if not in the dev tools) but that is a thing of the past: https://developer.mozilla.org/en/docs/Web/API/Console/log

I actually use console.error in some try/catch to log an error because given minification and loaders it is almost the only way to log an error and have the user (telling them to press F12) communicate the error to you. The only other way -and I have thought about this multiple times myself- is to implement as logging resource/service (restful I guess) and post errors to it via ajax. That could be fairly neat and relatively simple thing. Multiple web apps could make use of that.